test/api.js

```diff
@@ -24,4 +25,20 @@ suite('API Calls', function() {
     })
     .catch(done);
   });
+
+  test('Overrides', function(done) {
+    api.dlLoader()
+      .then(function() {
+        return api.install('ember', '1.13.2');
+      })
+      .then(function() {
+        return api.normalize('ember').then(function(normalized) {
+          var content = fs.readFileSync(normalized.replace('file://', ''), 'utf-8');
```

Comment:
In Windows, we need to remove three `///` here. Feel free to define a `fromFileURL` method next to https://github.com/jspm/jspm-cli/blob/master/lib/common.js#L24.